src/frame.rs

```diff
@@ -33,6 +33,7 @@ use crate::ranges;
 use crate::stream;
 
 pub const MAX_CRYPTO_OVERHEAD: usize = 8;
+pub const MAX_DGRAM_OVERHEAD: usize = 8;
```

Comment:
Also, FWIW, we are probably never going to use an 8-bytes varint to represent the frame length, so you can probably just use 2 here.

Reply:
agreed, this could in some conditions let us squeeze a few more dgrams in a packet too, so is a net win